app/controllers/concerns/csrf_protection.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# This module only gets mixed in to one place, but is that cleanest way to organize everything in one place related
+# to this responsibility alone.
```

Comment:
This comment might not age well.

I'd like to see a "why-style"  comment explaining how we're api-only and cannot use the normal rails `protect_from_forgery`.

Reply:
Updated.